src/lib/patchConceptForm.ts

```diff
@@ -17,7 +17,7 @@ export const patchConceptFromForm = (values, { concept, dispatch, lastPatchedRes
   dispatch(conceptPatchIsSavingAction(conceptId));
   patchConcept(conceptId, diff)
     .then(response => {
-      dispatch(conceptPatchSuccessAction(conceptId, values, response));
+      dispatch(conceptPatchSuccessAction(conceptId, !!values.status, response));
```

Comment:
Dette blir vel et problem om vi får mulighet til å avpublisere

Reply:
nei, vil bare vite når man patcher status, spiller ikke rolle hva status settes til. Men når denne er satt vil statusbar vise at status er endret.

Reply:
Ja, selvfølgelig. Status er jo type string, hadde plutselig for meg at det var en boolean